--- task-2.5.3/src/CMakeLists.txt.orig	2021-01-04 06:30:22.000000000 +0100
+++ task-2.5.3/src/CMakeLists.txt	2021-01-29 22:51:54.310374964 +0100
@@ -6,7 +6,7 @@
                      ${CMAKE_SOURCE_DIR}/src/libshared/src
                      ${TASK_INCLUDE_DIRS})
 
-add_library (task CLI2.cpp CLI2.h
+add_library (task STATIC CLI2.cpp CLI2.h
                   Context.cpp Context.h
                   DOM.cpp DOM.h
                   Eval.cpp Eval.h
@@ -27,7 +27,7 @@
                   sort.cpp
                   util.cpp util.h)
 
-add_library (libshared libshared/src/Color.cpp         libshared/src/Color.h
+add_library (libshared STATIC libshared/src/Color.cpp         libshared/src/Color.h
                        libshared/src/Configuration.cpp libshared/src/Configuration.h
                        libshared/src/Datetime.cpp      libshared/src/Datetime.h
                        libshared/src/Duration.cpp      libshared/src/Duration.h
