--- task-1.9.0-orig/configure.ac	2010-02-23 19:16:09.566235221 +0100
+++ task-1.9.0/configure.ac	2010-02-23 19:17:08.632721877 +0100
@@ -29,8 +29,8 @@
   CXXFLAGS="$CXXFLAGS -Wall -pedantic -ggdb3 -DDEBUG"
   AC_MSG_RESULT(yes)
 else
-  CFLAGS="$CFLAGS -Wall -pedantic -O3"
-  CXXFLAGS="$CXXFLAGS -Wall -pedantic -O3"
+  CFLAGS="$CFLAGS -Wall -pedantic"
+  CXXFLAGS="$CXXFLAGS -Wall -pedantic"
   AC_MSG_RESULT(no)
 fi
 
